### PR TITLE
 update python/django support: add django 2.2

### PR DESCRIPTION
**Pull request by MorganeAD**

### CODE REVIEW

**Review by hsmett — APPROVED** [review]
lgtm

**Comment by mrunge** [comment]
This fails in tox:
```
&cache=shared')...
SystemCheckError: System check identified some issues:

ERRORS:
?: (admin.E410) 'django.contrib.sessions.middleware.SessionMiddleware' must be in MIDDLEWARE in order to use the admin application.

System check identified 1 issue (0 silenced).
ERROR: InvocationError for command '/home/mrunge/tmp/django-appconf/.tox/py35-dj22/bin/coverage run /home/mrunge/tmp/django-appconf/.tox/py35-dj22/bin/django-admin.py test -v2 tests' (exited with code 1)
```

File: tests/test_settings.py [hunk 1]
```diff
@@ -14,9 +14,29 @@
...
     'django.contrib.auth',
     'django.contrib.admin',
+    'django.contrib.messages',
+    'django.contrib.sessions',
     'tests',
 ]
 
+MIDDLEWARE = [
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django.contrib.messages.middleware.MessageMiddleware",
```
Comment (mrunge) [inline tests/test_settings.py:24]: also add "django.contrib.sessions.middleware.SessionMiddleware",

**Review by mrunge — CHANGES_REQUESTED** [review]
Could you please add this as well? This makes tox pass for me.

**Comment by hsmett** [comment]
E410 was added in django 2.2.1, which is the reason why travis didn't complain at the time.

[https://docs.djangoproject.com/en/2.2/releases/2.2.1/](https://docs.djangoproject.com/en/2.2/releases/2.2.1/)

**Comment by mrunge** [comment]
Right. Thank you for the context. 

This patch (plus the additional line) made django-appconf build and pass tests during build on Fedora infrastructure.

**Review by mrunge — APPROVED** [review]
Thank you, this looks good to me!

**Review by carltongibson — APPROVED** [review]
OK, super, looks good. Thank you.

**Comment by MorganeAD** [comment]
thanks ^^

**Pull request merged**